classes/log/store.php

```diff
@@ -53,12 +54,21 @@ class store extends php_obj implements log_writer {
 
     protected $loggingenabled = false;
 
+    /** @var bool $logguests true if logging guest access */
+    protected $logguests;
+
+    /** @var array $routes An array of routes to include */
+    protected $routes = array();
+
     /**
      * Constructs a new store.
      * @param log_manager $manager
      */
     public function __construct(log_manager $manager) {
         $this->helper_setup($manager);
+        $this->logguests = $this->get_config('logguests', 1);
```

Comment:
@ryansmith94 It seems to be pretty standard Moodle syntax, if not ugly and against our coding standards for the rest of the plugin. I've never logged guest access, so I don't know the details. From my minor bit of research, it does seem standard, and is in the database logstore plugin exactly like this.